spaceMissionManager-angular/src/main/webapp/views/spacecrafts.html

```diff
@@ -1 +1,77 @@
-<h1>Spacecrafts</h1>
\ No newline at end of file
+<h1>Spacecrafts</h1>
+<div ng-if="!edit && !create">
+    <table class="table table-striped">
+        <tr>
+            <th>Name</th>
+            <th>Type</th>
+            <th></th>
+            <th></th>
+
+        </tr>
+        <tr ng-repeat="spacecraft in spacecrafts">
+            <td>{{spacecraft.name}}</td>
+            <td>{{spacecraft.type}}</td>
```

Comment:
<td>
                <div ng-repeat="cc in spacecraft.craftComponents track by $index">
                    {{cc.name}} - {{cc.readyToUse?"YES":"NO"}}
                </div>
</td>